lib/ae_mdw_web/controllers/transfer_controller.ex

```diff
@@ -1,4 +1,5 @@
 defmodule AeMdwWeb.TransferController do
+  # credo:disable-for-this-file
```

Comment:
This isn't needed and only happens because it's branched-off an old master version

Reply:
ci complained, that's why it's disabled (for a file that is not part of PR).

Reply:
it complained because of the reason I previously mentioned, which is fixed by rebasing with master

Reply:
done, away of AFK ...